### PR TITLE
HAWQ-1320. Remove reference to PXF rpm version in source

### PR DESCRIPTION
**Pull request by shivzone**

### CODE REVIEW

File: pxf/Makefile [hunk 1]
```diff
@@ -38,6 +38,10 @@ ifneq "$(VENDOR)" ""
     BUILD_PARAMS+= -Pvendor="$(VENDOR)"
 endif
 
+ifneq "$(PXF_VERSION)" ""
+    BUILD_PARAMSi+= -Pversion="$(PXF_VERSION)"
```
Comment (ljainpivotalio) [inline pxf/Makefile]: There is an extra i after BUILD_PARAMS.

**Comment by ljainpivotalio** [comment]
Please feel free to merge when done

File: pxf/Makefile (hunk 1 above)
Comment (shivzone) [inline pxf/Makefile]: good catch

File: pxf/Makefile [hunk 2]
```diff
@@ -38,6 +38,10 @@ ifneq "$(VENDOR)" ""
     BUILD_PARAMS+= -Pvendor="$(VENDOR)"
 endif
 
+ifneq "$(PXF_VERSION)" ""
```
Comment (sansanichfb) [inline pxf/Makefile]: Shouldn't we fail if there is no value for PXF_VERSION available?

Comment (shivzone) [inline pxf/Makefile]: Not really. We should still be able to run simple dev builds which don't require PXF version

**Pull request closed without merging**